src/utils/hooks/useListApplications.tsx

```diff
@@ -18,12 +15,6 @@ const useListApplications = (
   graphQLFilterObject?: object
 ) => {
   const FILTER_DEFINITIONS = useGetFilterDefinitions()
-  const [applications, setApplications] = useState<ApplicationListShape[]>([])
```

Comment:
We had a lot of unnecessary `useState` variables, so have removed quite a few and just returned the state directly from the Apollo hook.